clash-cores/src/Clash/Cores/Xilinx/Floating/Annotations.hs

```diff
@@ -99,40 +93,36 @@ veriBinaryPrim
   -> String
   -> Primitive
 veriBinaryPrim primName tclTFName funcName =
-  InlinePrimitive [Verilog, SystemVerilog] $ unindent [i|
-    [ { "BlackBox" :
-        { "name"      : "#{show primName}"
-        , "type"      :
-    "#{show primName}
-      :: ( KnownDomain dom           --     ARG[0]
-         , KnownNat d                --     ARG[1]
-         , HasCallStack              --     ARG[2]
-         )
-      => Config                      --     ARG[3]
-      -> Clock dom                   --     ARG[4]
-      -> Enable dom                  --     ARG[5]
-      -> DSignal dom n Float         -- x , ARG[6]
-      -> DSignal dom n Float         -- y , ARG[7]
-      -> DSignal dom (n + d) Float"
-      , "kind"      : "Declaration"
-      , "template"  :
-    "~DEVNULL[~LIT[3]]~INCLUDENAME[0] ~GENSYM[#{funcName}][0] (
-      .aclk(~ARG[4]),
-    ~IF~ISACTIVEENABLE[5]~THEN  .aclken(~ARG[5]),
-    ~ELSE~FI  .s_axis_a_tvalid(1'b1),
-      .s_axis_a_tdata(~ARG[6]),
-      .s_axis_b_tvalid(1'b1),
-      .s_axis_b_tdata(~ARG[7]),
-      .m_axis_result_tvalid(),
-      .m_axis_result_tdata(~RESULT)
-    );"
-        , "includes"  : [ {"extension": "tcl"
-                          ,"name": "floating_point"
-                          ,"format": "Haskell"
-                          ,"templateFunction": "#{show tclTFName}"
-                          }
-                        ]
-        }
-      }
-    ]
-    |]
+  InlineYamlPrimitive [Verilog, SystemVerilog] [__i|
+  BlackBox:
```

Comment:
I normally wouldn't comment on this, but given that this is in the diff anyway: the other blackboxes in this patch indent like this:

```suggestion
  InlineYamlPrimitive [Verilog, SystemVerilog] [__i|
    BlackBox:
```

which I like better.

Reply:
Right, if you look at its brother [here](https://github.com/clash-lang/clash-compiler/pull/2326/files/cfc24e2b27d972d12c572ea94efb665b050fceb5#diff-73ec9a585ecf605d2db31b5b6e420ccdfbea78007357c3f4a365529f66b13b03R30) you'll see that one was not intended, thanks for spotting it!
```haskell
vhdlBinaryPrim primName tclTFName funcName = InlineYamlPrimitive [VHDL] [__i|
  BlackBox:
```